src/middlewares/prometheus_middleware.js

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+const excludedPaths = ['/health', '/metrics'];
+
+function prometheusMiddleware(promClient, register) {
```

Comment:
We generally prefer to use arrow functions as much as possible. In this `function prometheusMiddleware(promClient, register) {` => `const prometheusMiddleware = (promClient, register) => {`